src/main/java/org/wildfly/security/auth/realm/ldap/LdapSecurityRealmBuilder.java

```diff
@@ -205,6 +205,7 @@ private void assertNotBuilt() {
         private List<AttributeMapping> attributes = new ArrayList<>();
         private LdapName newIdentityParent = null;
         private Attributes newIdentityAttributes = null;
+        private String filterName;
```

Comment:
Is this the same value as "nameFilter" up above?  If so, they should have the same name.